src/app.js

```diff
@@ -37,5 +37,30 @@ app.get('/online-users', async (req, res) => {
   }
 })
 
+app.post('/create-game', async (req, res) => {
+  try {
+    const { users } = req.body
+    await dynamo.createGame(users)
+    return res.status(200).send()
+  } catch (e) {
+    console.error('Error while creating game 😬', e)
+    res.status(500).send({ message: e.message })
+  }
+})
+
+app.get('/my-started-game', async (req, res) => {
+  try {
+    const { userId } = req.query
+    const startedGames = await dynamo.getStartedGames(userId)
+    const gamesId = startedGames.map(({ id: gameId }) => {
+      return { gameId }
+    })
+    res.send(gamesId)
```

Comment:
`getStartedGames` returns an empty array if no games are found, and we still return a OK response in that case. We should return 404 if no games are found.